frontend/src/components/Table/Tab.tsx

```diff
@@ -94,17 +132,46 @@ const Tab: React.FC<TabProps> = ({ resourceLabel }) => {
                 type="primary"
                 icon={<PlusOutlined />}
                 onClick={handleAdd}
-                style={{ marginBottom: 16 }}
+                size="large"
+                style={{
+                    position: "fixed",
+                    bottom: "16px",
+                    right: "16px",
+                    zIndex: 1000,
+                    borderRadius: "50px",
+                    padding: "0 16px",
+                }}
             >
                 Add
             </Button>
```

Comment:
I don't think we need it to say add. Giant plus would be enough (button would need to be a little bigger). Some other opinions would be great. Another thing, it should be higher. Right now it covers pagination.

![image](https://github.com/user-attachments/assets/56760c74-0701-44e3-bc54-a1c761f93096)